File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    var answer = [];
    
    let lux=51, luy=51, rdx=-1, rdy=-1;
    wallpaper.forEach((s, index) => {
        
        for(let i=0; i<s.length; i++){
            if(s[i] === '#'){
                lux = Math.min(lux, index);
                luy = Math.min(luy, i);
                rdx = Math.max(rdx, index+1);
                rdy = Math.max(rdy, i+1);
            }
        }
    });
    
    answer.push(lux);
    answer.push(luy);
    answer.push(rdx);
    answer.push(rdy);
    
    return answer;
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<string> wallpaper) {
    vector<int> answer;
    
    int lux=51, luy=51, rdx=-1, rdy=-1;
    
    for(int i=0; i<wallpaper.size(); i++){
        for(int j=0; j<wallpaper[i].length(); j++){
            if(wallpaper[i][j] == '#'){
                lux = min(lux, i);
                luy = min(luy, j);
                rdx = max(rdx, i+1);
                rdy = max(rdy, j+1);
            }      
        }
    }
    
    answer.push_back(lux);
    answer.push_back(luy);
    answer.push_back(rdx);
    answer.push_back(rdy);
    
    return answer;
}